i18n/de/docusaurus-plugin-content-docs/current/toolbox/accessibility_indicators/oev_gueteklassen.md

```diff
@@ -6,126 +6,120 @@ import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
 
 ![ÖV-Güteklassen in GOAT](/img/toolbox/accessibility_indicators/gueteklassen/example.png "ÖV-Güteklassen in GOAT")
 
-:::info 
-The calculation of the public transport quality classes is only available for areas where the transport network is integrated into GOAT.
+:::info
+Die Berechnung der ÖV-Güteklassen ist nur für Gebiete verfügbar, in denen das Verkehrsnetz in GOAT integriert ist.
 
-<div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
-  <img src={require('/img/toolbox/accessibility_indicators/gueteklassen/geofence-pt.png').default} alt="Geofence for ÖV-Güteklassen calculation in GOAT" style={{ maxHeight: "400px", maxWidth: "400px", alignItems:'center'}}/>
-</div> 
+<div style="display: flex; flex-direction: column; align-items: center;">
+  <img src={require('/img/toolbox/accessibility_indicators/gueteklassen/geofence-pt.png').default} alt="Geofence für ÖV-Güteklassen Berechnung in GOAT" style="max-height: 400px; max-width: 400px; align-items: center;"/>
+</div>
 
-In case you need to perform analysis beyond this geofence, feel free to contact the [Support](https://plan4better.de/en/contact/ "Contact Support") and we will check what is possible. 
+Falls Sie eine Analyse außerhalb dieses Geofence durchführen müssen, kontaktieren Sie bitte den [Support](https://plan4better.de/en/contact/ "Support kontaktieren") und wir prüfen, was möglich ist.
 :::
 
-## 2. Example use cases
+## 2. Anwendungsbeispiele
 
-- How good is public transport supply in different parts of the city?
-- How many people are underserved by public transport? Where is the need for further supply?
-- How does the quality of public transport services differ at different times of the week and day?
+- Wie gut ist das Angebot an öffentlichen Verkehrsmitteln in verschiedenen Teilen der Stadt?
+- Wie viele Menschen sind unterversorgt mit öffentlichen Verkehrsmitteln? Wo besteht Bedarf an weiteren Angeboten?
+- Wie unterscheidet sich die Qualität der öffentlichen Verkehrsmittel zu verschiedenen Zeiten der Woche und des Tages?
 
-## 3. How to use the indicator?
+## 3. Verwendung des Indikators
 
 <div class="step">
   <div class="step-number">1</div>
-  <div class="content">Click on <code>Toolbox</code> <img src={thematicIcon} alt="toolbox" style={{width: "25px"}}/>. </div>
+  <div class="content">Klicken Sie auf <code>Toolbox</code> <img src={thematicIcon} alt="toolbox" style="width: 25px;"/>.</div>
```

Comment:
In some of the other docs, I see a few places where "Click on" has not been translated to "Klicken Sie auf" like here. Could you do a quick check to replace it in the other files too?